video_xblock/static/js/studio_edit.js

```diff
@@ -407,5 +412,35 @@ function StudioEditableXBlock(runtime, element) {
     $().ready(function(){
         disableOption();
     });
+
+    var parseRelativeTime = function (value) {
+        // This function ensure you have two-digits
+        // By default max value of RelativeTime field on Backend is 23:59:59,
+        // that is 86399 seconds.
+        var maxTimeInSeconds = 86399;
+        var pad = function (number) {
+                return (number < 10) ? "0" + number : number;
```

Comment:
I've just wanted to return 2 characters mandatory. Converting number > 9 to string is an extra action